web/context.go

```diff
@@ -663,3 +663,15 @@ func (c *Context) RequireBotUserId() *Context {
 	}
 	return c
 }
+
+func (c *Context) RequireInvoiceId() *Context {
+	if c.Err != nil {
+		return c
+	}
+
+	if len(c.Params.InvoiceId) != 27 {
```

Comment:
Is this length guaranteed for all stripe invoice ids?

Reply:
In the Stripe API is not set but by now is the length we have so we can verify its a valid id